src/cloud/integrations/bitbucket-integration.md

```diff
@@ -164,13 +164,17 @@ To create an OAuth consumer in Bitbucket:
 
 ### Add a webhook in BitBucket
 
-In order to communicate events—such as a push—with your Cloud Git server, you need to create a webhook for your BitBucket repository.
+In order to communicate events—such as a push—with your Cloud Git server, is it necessary to have a webhook for your BitBucket repository. The method of setting up a Bitbucket integration detailed on this page, when followed correctly, automatically creates a webhook. It is important to verify the webhook to avoid creating multiple integrations.
```

Comment:
```suggestion
In order to communicate events—such as a push—with your Cloud Git server, must have a webhook for your BitBucket repository. The method of setting up a Bitbucket integration detailed on this page, when followed correctly, automatically creates a webhook. It is important to verify the webhook to avoid creating multiple integrations.
```